src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap';
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
 body {
+  padding: 0;
   margin: 0;
 }
+
+.card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    margin-inline: 3px;
```

Comment:
According to the code style rules, 'margin-inline' is not a standard property. Use 'margin-left' and 'margin-right' for better compatibility and readability.